RevenueCatUI/PaywallView.swift

```diff
@@ -80,6 +80,10 @@ public struct PaywallView: View {
                         .transition(Self.transition)
                         .task {
                             do {
+                                guard Purchases.isConfigured else {
+                                    throw PaywallError.purchasesNotConfigured
+                                }
```

Comment:
@aboedo handled this too, so no more crashes from `Purchases` not being configured :)